osu_beatmap_downloader/downloader.py

```diff
@@ -4,14 +4,43 @@
 import re
 import sys
 import time
+import win32api 
+import win32con
+from getpass import getuser
 
 import httpx
 from loguru import logger
 from PyInquirer import prompt
 
 # Windows
 if sys.platform.startswith("win32"):
-    DOWNLOAD_PATH = os.path.join(os.getenv("LOCALAPPDATA"), "osu!", "Songs")
+    key = win32api.RegOpenKey(win32con.HKEY_CLASSES_ROOT,
```

Comment:
As already mentioned reading the registry sounds overkill.

But hypothetically if you want to do it, you should start naming your variables better and start using idiomatic python code like this:

```Python
from pathlib import Path


if sys.platform.startswith("win32"):
    osu_reg_key = win32api.RegOpenKey(win32con.HKEY_CLASSES_ROOT, "osu\\DefaultIcon", 0, win32con.KEY_READ)
    osu_reg_val = win32api.RegQueryValue(osu_reg_key, "")
    osu_exe_path, _ = osu_reg_val.split(",")
    osu_exe_path = osu_exe_path.strip("\"")
    osu_install_path = Path(osu_exe_path).parent
    osu_config_path = osu_install_path / f"osu!.{getuser()}.cfg"

    with open(osu_config_path, encoding="utf-8", errors="ignore") as config_file:
        for line in config_file:
            if line.startswith("BeatmapDirectory"):
                key, value = line.split(" = ")
                DOWNLOAD_PATH = osu_install_path / value
                break

    USERPROFILE = os.getenv("USERPROFILE")
```